run.sh

```diff
@@ -47,7 +45,10 @@ fi
 if [ -z "$COLLECTD_FLUSHINTERVAL" ]; then
 	COLLECTD_FLUSHINTERVAL=$COLLECTD_INTERVAL
 fi
-
+if [ ! -S /var/run/docker.sock ]; then
+    echo "The docker socket was not mounted to the container, the collectd docker plugin will be disabled"
```

Comment:
I don't believe that collectd is capitalized like that. Usage everywhere by the community and within SignalFx is "collectd" not "CollectD"